src/main/webapp/scripts/map.js

```diff
@@ -111,6 +118,8 @@ function addMarker(map) {
   }
 }
 
+let addedMarker;
```

Comment:
Is this unused? Please remove in case it is!

Reply:
Yes, thank you for spotting it